courses/free-courses-it.md

```diff
@@ -58,6 +59,11 @@
 * [Assembler x86](https://www.youtube.com/playlist?list=PLUJjY3hQLJ3NHQ9315KVvgiZ3v2FLtcbs)
 
 
+### Bash / Shell
+
+* [Programmazione di sistema - Bash](https://www.youtube.com/playlist?list=PLhlcRDRHVUzR-5TKDC1VPMtyhEyyQ5uwy) - N. Bicocchi, Università degli studi di Modena e Reggio Emilia
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

Reply:
ok